skbio/io/format/embl.py

```diff
@@ -0,0 +1,1497 @@
+"""
+EMBL format (:mod:`skbio.io.format.embl`)
+=========================================
+
+.. currentmodule:: skbio.io.format.embl
+
+EMBL format stores sequence and its annotation together. The start of the
+annotation section is marked by a line beginning with the word "ID". The start
+of sequence section is marked by a line beginning with the word "SQ" and the
+end of the section is marked by a line with only "//". More information on
```

Comment:
more precisely, `//` denote the end of embl record. Maybe it need to be described also for genbank objects?